### PR TITLE
10.0 fix document module call

### PR DESCRIPTION
**Pull request by john-botella**
I had not seen but it is also necessary to modify the file path 
https://github.com/Dolibarr/dolibarr/pull/12715

### CODE REVIEW

File: htdocs/core/class/html.formfile.class.php [hunk 1]
```diff
@@ -679,7 +679,7 @@ public function showdocuments($modulepart, $modulesubdir, $filedir, $urlsource,
 						$modulepart=$tmp[0];
 						$submodulepart=$tmp[1];
 					}
-			        $file=dol_buildpath('/'.$modulepart.'/core/modules/'.$modulepart.'/modules_'.$submodulepart.'.php', 0);
+			        $file=dol_buildpath('/'.$modulepart.'/core/modules/'.$submodulepart.'/modules_'.$submodulepart.'.php', 0);
```
Comment (eldy) [inline htdocs/core/class/html.formfile.class.php]: We should have only one dir into core/modules per module. And we can put several "modules_submodulepart" when we need several handler.
What make you think you need to change this too ?

Comment (atm-john) [reply htdocs/core/class/html.formfile.class.php]: imagine you have a module call MyModule with 3 objects with documents : MyModule, ObjectA and ObjectB

/MyModule/core/modules/MyModule/modules_MyModule.php
/MyModule/core/modules/objecta/modules_objecta.php
/MyModule/core/modules/objectb/modules_objectb.php

Does I misunderstand  something ?

Comment (eldy) [reply htdocs/core/class/html.formfile.class.php]: If you have ObjectA and ObjectB, you should get
/MyModule/core/modules/mymodule/modules_objecta.php
/MyModule/core/modules/mymodule/modules_objectb.php
and not 
/MyModule/core/modules/objecta/modules_objecta.php
/MyModule/core/modules/objectb/modules_objectb.php

You can have a look at modulebuilder tree to have confirmation.

Comment (atm-john) [reply htdocs/core/class/html.formfile.class.php]: Ok

**Pull request closed without merging**